services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -238,9 +238,10 @@ public String getType() {
      * Instantiates a new space event info.
      *
      * @param socialSpace the social space
+     * @param spaceIdentityId the space identity id
      */
-    public SpaceEventInfo(Space socialSpace) {
-      super(socialSpace.getPrettyName(), socialSpace.getDisplayName());
+    public SpaceEventInfo(Space socialSpace, String spaceIdentityId) {
```

Comment:
Why do we need to keep both ? could we remove socialSpace attribute ?

Reply:
IMO we can't remove it, because we need the space group id.

Reply:
We cna get the groupID by getting the space with its identity  ID